html-syntax/index.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>HTML Syntax</title>
+  </head>
+  <body>
+    <img src="https://www.dexerto.com/wp-content/uploads/2020/08/pokemon-pokedex-entry-hints-at-dragonite-in-sword-and-shield.jpg" alt="a flying dragonite" width="500" height="300">
+    <br>
+    Dragonite <br>
+    &copy; Pokemon. &trade; Nintendo.
```

Comment:
Instead of using `br` elements, you should put each of these lines into their own `p` element. You'll get the same effect, and it will be more semantic HTML, since every bit of text will be in an element, rather than free-floating in the `body` element.

Reply:
thanks for the correction!

Reply:
![image](https://user-images.githubusercontent.com/30616230/129634722-febddd5f-56e2-4c38-bc9b-ad16545c76f2.png)